config/drs_objects_schema.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "DRS Compliance Suite DRS Objects Schema Config",
+    "description": "DRS Compliance Suite Config v1.2",
+    "type": "object",
+    "$ref": "drs_objects_schema.json#/definitions/drs_object",
+    "definitions": {
+        "drs_object": {
+            "type": "object",
+            "properties": {
+                "drs_id": {
+                    "type": "string",
+                    "description": "The DRS ID of the object"
+                },
+                "expected_status_code": {
+                    "type": "integer",
+                    "description": "The expected status code for testing the DRS object. 200 by default if not provided."
+                },
+                "has_auth_get_object": {
+                    "type": "boolean",
+                    "description": "True if get object endpoint requires auth info. False by default if not provided."
```

Comment:
It would be nice if we could highlight that when this field is set to True, it implies that the auth_info used for the DRS object endpoint will be the same as the auth_info used for the access endpoint.